File: ProgramasJava/src/br/com/geekuniversity/secao07/Exercicio01.java
package br.com.geekuniversity.secao07;

import java.util.*;

public class Exercicio01 {

	public static void main(String[] args) {
		
		int n, m = 0;
		Scanner tela = new Scanner(System.in);
		
		System.out.println("Informe um nº qualquer: ");
		n = tela.nextInt();
		
		while (n != 0) {
			
			if (n > m) {
				
				m = n;
				
			}
			
			System.out.println("Informe um nº qualquer: ");
			n = tela.nextInt();
			
		}
		
		System.out.printf("%nO maior nº digitado foi %d.", m);
		
		tela.close();
		
	}

}

File: ProgramasPython/secao07/Exercicio01.py
maior = 0;
tela = int(input("Insira um nº na tela: "));

while tela != 0:
    
    if tela > maior:
        
        maior = tela;
        
    tela = int(input("Insira um nº na tela: "));

print("O maior nº é {0}.".format(maior));